src/NServiceBus.Transport.RabbitMQ/Configuration/ConnectionConfiguration.cs

```diff
@@ -9,8 +9,10 @@
     class ConnectionConfiguration
     {
         const bool defaultUseTls = false;
-        const int defaultPort = 5672;
-        const int defaultTlsPort = 5671;
+        const int defaultBrokerPort = 5672;
```

Comment:
I don't think any of the changes in this file should be done. Instead, the management client should just take a regular `ConnectionConfiguration` instance. The knowledge of needing to override the ports for the management client should be there instead.